runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaPlugin.java

```diff
@@ -239,6 +239,7 @@ private void createTimer(Duration duration)
 
 		BufferedImage image = itemManager.getImage(ItemID.ENSOULED_DEMON_HEAD);
 		infoBoxManager.addInfoBox(new AggressionTimer(duration, image, this));
+		infoBoxManager.removeIf(UncalibratedInfobox.class::isInstance);
```

Comment:
This is fine for normal operation on a single account, but would lead to the infobox not reappearing when logging out of one account and logging in to another account which didn't have their areas set up.